TADashboard/src/test/resources/suites/test.xml

```diff
@@ -8,7 +8,7 @@
     <test name="Run Chrome Local">
```

Comment:
create your own suite instead of updating the test one.

Reply:
Yeah I got it.

Reply:
you still update it

Reply:
No. I use my xml file. It's name DA_DP_TC66